tests/playTests.sh

```diff
@@ -409,6 +409,34 @@ println "compress multiple files including a missing one (notHere) : "
 $ZSTD -f tmp1 notHere tmp2 && die "missing file not detected!"
 
 
+println "\n===>  size-hint mode"
+
+./datagen -g11000 > tmp
+println "test : basic file compression vs streaming compression vs hinted streaming compression"
+$ZSTD -14 -f tmp -o tmp.zst 2>&1 | tee file.out
+cat tmp | $ZSTD -14 -f -o tmp.zst  # only run for convenience of comparison
+cat tmp | $ZSTD -14 -f -o tmp.zst --size-hint=11000 2>&1 | tee stream_sized.out
+
+file_ratio=$(cat file.out | awk '{print $4}' | sed 's/%//g')
+stream_sized_ratio=$(cat stream_sized.out | awk '{print $4}' | sed 's/%//g')
+rm file.out stream_sized.out
+
+ratio_diff=$(echo $stream_sized_ratio - $file_ratio | bc)
```

Comment:
Sadly, we can't rely in `bc` being present, and we want these tests to work on Windows, OS X, and bare bones linux. It looks like it isn't present on Circle-CI.

Can test that providing the size hint is smaller than the file compression? It should hold because we don't write the content size, so we should compress 1+ bytes smaller.

```
file_size=$($ZSTD -14 tmp -c | wc -c)
stream_size=$(cat tmp | $ZSTD -14 | wc -c)
if [ "$stream_size" -ge "$file_size" ]; then
  die "message"
fi
```